src/symbol/shaping.js

```diff
@@ -210,12 +215,12 @@ function shapeText(text: Formatted,
         right: translate[0],
         writingMode,
         lineCount: lines.length,
-        yOffset: -17 // the y offset *should* be part of the font metadata
+        yOffset: -17, // the y offset *should* be part of the font metadata
```

Comment:
Since `yOffset` is not provided from the font metadata per glyph, it could be removed from this structure and the value hard-coded separately for cases where baseline data is not available.